py/desiutil/dust.py

```diff
@@ -526,12 +526,12 @@ def __init__(self, fname, scaling):
         self.lam_scal = header['LAM_SCAL']
         self.sign = header['LAM_NSGP']  # north = 1, south = -1
 
-    def ebv(self, l, b, interpolate):
+    def ebv(self, gal_l, gal_b, interpolate):
```

Comment:
I like `gal_l` and `gal_b` better than just `l`, and `b`, but this is an API change that could break client code.  Please grep desispec, desitarget, desisurvey, and tutorials for uses of this function to make sure that they aren't using `ebv(l=..., b=...)`.  Formally this change would require a 4.0 tag, so let's consider undoing it here since it is stylistic, and save it for the 4.0 changes that aren't API backwards compatible.

Reply:
OK, I'll revert that.

Reply:
Reverted, and #213 created to track this for 4.0.